modules/deploy/pages/deployment-option/self-hosted/docker-image.adoc

```diff
@@ -3,7 +3,7 @@
 :page-context-links: [{"name": "Docker", "to": "deploy:deployment-option/self-hosted/docker-image.adoc" },{"name": "Kubernetes", "to": "deploy:deployment-option/self-hosted/kubernetes/k-deploy-connectors.adoc" } ]
 :page-toclevels: 1
 
-This topic describes how to use the https://hub.docker.com/r/redpandadata/connectors/tags[Redpanda Connectors Docker image^] to configure and deploy connectors for Redpanda. The image contains:
+This topic describes how to use the https://hub.docker.com/r/redpandadata/connectors/tags[Redpanda Connectors Docker image^] to configure and deploy connectors for Redpanda. This configuration requires you to configure and use https://redpanda.com/guides/kafka-tutorial/what-is-kafka-connect[Kafka Connect]. https://docs.redpanda.com/current/deploy/deployment-option/cloud/managed-connectors/[Managed Connectors] are available only in Redpanda Cloud. The Docker image contains:
```

Comment:
```suggestion
This topic describes how to use the https://hub.docker.com/r/redpandadata/connectors/tags[Redpanda Connectors Docker image^] to configure and deploy connectors for Redpanda. This configuration requires  https://redpanda.com/guides/kafka-tutorial/what-is-kafka-connect[Kafka Connect]. https://docs.redpanda.com/current/deploy/deployment-option/cloud/managed-connectors/[Managed Connectors] are available only in Redpanda Cloud. The Docker image contains:
```
suggestion, because there was a lot of "configuration"

Reply:
Suggest changing the link to the managed connectors doc to a [glossterm link for connector](https://deploy-preview-245--redpanda-docs-preview.netlify.app/current/reference/glossary/#connector). There's 3 links in that intro paragraph, and there would still be 3 links... but with they hover text, they might not feel the need for a link to more info.